packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -61,15 +89,9 @@ export const BootstrapCommand: ICommand = {
 
     // Assure all kbn projects with bin defined scripts
     // copy those scripts into the top level node_modules folder
-    await linkProjectExecutables(projects, projectGraph);
-
-    // Bootstrap process for Bazel packages
     //
-    // NOTE: Bazel projects will be introduced incrementally
-    // And should begin from the ones with none dependencies forward.
-    // That way non bazel projects could depend on bazel projects but not the other way around
-    // That is only intended during the migration process while non Bazel projects are not removed at all.
-    await runBazel(['build', '//packages:build']);
+    // NOTE: We don't probably need this anymore, is actually not being used
```

Comment:
Then can we remove it? ;-)

Reply:
That does not belong to that PR, I've just moved it around. In order to remove this I will have the need to investigate it better. I think we are not currently using it in any package right now but that is the piece of code that allows for a package that declares a bin script to get it used locally. I believe its better to not touch it as part of this PR